lib/fullstory/index.js

```diff
@@ -67,6 +68,11 @@ FullStory.prototype.identify = function(identify){
     del(traits, 'name');
   }
 
+  if (traits.Email) {
```

Comment:
That's just from the test though, since you're passing in `Email`. Users should be passing in `email`. This is cleaning up behind users unnecessarily. I'd drop the if check and just do:

```
traits.email = identify.email().toLowerCase();
```

cc @ndhoule @harrietgrace for review on this as well.